graviti/openapi/data.py

```diff
@@ -255,7 +255,7 @@ def update_data(
         data: The update data.
 
     Examples:
-        >>> update_dataset(
+        >>> update_data(
```

Comment:
commit message is not clear, should we mention "correct the function name in example"?